Modules/01_oop_and_ood/src/Triangle.ts

```diff
@@ -0,0 +1,44 @@
+import { Shape } from './Shape';
+import { Point } from './Point';
+
+export class Triangle extends Shape {
+
+  constructor(a: Point, b: Point, c: Point, color?: string, filled?: boolean) {
+    super([a, b, c], color, filled);
+  }
+
+  toString(): string {
+    return `Triangle[v1=${this.points[0].toString()},v2=${this.points[1].toString()},v3=${this.points[2].toString()}]`;
+  }
+
+  getType(): string {
+    const s1: number = Math.round((this.points[0].distance(this.points[1])) * 100) / 100;
```

Comment:
Why do the multiplication and division?

Reply:
for roundup to 2 decimals `Math.round(num * 100) / 100`

Reply:
Have you considered the `toFixed` method?

Reply:
yes, but I wanted to keep `types` intouched, basically to work with numbers :) I agree, that `toFixed` is `cleaner` solution :)

Reply:
You can wrap it in `parseFloat` to keep it as a number :)